### PR TITLE
feat(Pagination): remove `is` prefix in props

### LINKED ISSUE #3492: Rename `props`: `is*` eliminates all `is` prefix in the `Pagination` package
### Description

Change and unify the names of props.
Removing all is prefix from is*

### Problem Statement/Justification

For example, `isOpen` becomes `open` and `isDisabled` becomes `disabled`.

### Proposed Solution or API

props to be changed are marked `@deprecated` in v1

### Alternatives

_No response_

### Are you willing to participate in implementing this feature and create a pull request include the implementation?

None

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by kaehehehe**
Closes #3492

## Description

Removing all is prefix from is*

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

File: packages/components/pagination/src/pagination-item.tsx [hunk 1]
```diff
@@ -27,12 +39,14 @@ interface PaginationItemOptions {
    * If `true`, the pagination item will be activated.
    *
    * @default false
+   * @deprecated Use `active` instead.
```
Comment (108yen) [inline packages/components/pagination/src/pagination-item.tsx:42]: Spacing one line apart.

File: packages/components/pagination/src/pagination-item.tsx [hunk 2]
```diff
@@ -27,12 +39,14 @@ interface PaginationItemOptions {
...
    */
   isActive?: boolean
   /**
    * If `true`, the pagination item will be disabled.
    *
    * @default false
+   * @deprecated Use `disabled` instead.
```
Comment (108yen) [inline packages/components/pagination/src/pagination-item.tsx:49]: Spacing one line apart.

File: packages/components/pagination/src/use-pagination.ts [hunk 3]
```diff
@@ -40,6 +40,13 @@ export interface UsePaginationProps {
    *
    * @default false
    */
+  disabled?: boolean
+  /**
+   * If `true`, the pagination all item will be disabled.
+   *
+   * @default false
+   * @deprecated Use `disabled` instead.
```
Comment (108yen) [inline packages/components/pagination/src/use-pagination.ts:48]: Spacing one line apart.

File: packages/components/pagination/src/pagination-item.tsx
Comment (108yen) [inline packages/components/pagination/src/pagination-item.tsx:1]: `PaginationItem` is not exported in `index.ts`. So I think you can remove is prefix props.

**Review by 108yen — CHANGES_REQUESTED** [review]

**Review by 108yen — COMMENTED** [review]
No changeset.

**Comment by kaehehehe** [comment]
> No changeset.

I determined that it was unnecessary because there were no changesets in the already merged PRs. @108yen 

- https://github.com/yamada-ui/yamada-ui/pull/3418
- https://github.com/yamada-ui/yamada-ui/pull/3420

**Comment by 108yen** [comment]
@kaehehehe 
The change will not be published without changeset.

File: packages/components/pagination/src/pagination-item.tsx
Comment (kaehehehe) [reply packages/components/pagination/src/pagination-item.tsx:1]: Please read this. https://github.com/yamada-ui/yamada-ui/issues/3421

File: packages/components/pagination/src/pagination-item.tsx (hunk 1 above)
Comment (kaehehehe) [reply packages/components/pagination/src/pagination-item.tsx:43]: Fixed.

File: packages/components/pagination/src/pagination-item.tsx (hunk 2 above)
Comment (kaehehehe) [reply packages/components/pagination/src/pagination-item.tsx:51]: Fixed.

File: packages/components/pagination/src/use-pagination.ts (hunk 3 above)
Comment (kaehehehe) [reply packages/components/pagination/src/use-pagination.ts:49]: Fixed.

**Comment by kaehehehe** [comment]
> @kaehehehe The change will not be published without changeset.

[This PR](https://github.com/yamada-ui/yamada-ui/pull/3491/files) was approved even though it doesn't have a changeset. Why is it that only my PR requires a changeset?

**Comment by 108yen** [comment]
@kaehehehe 
I think the previous PR is wrong.

**Review by 108yen — APPROVED** [review]

**Review by hirotomoyamada — APPROVED** [review]

**Pull request merged**